### PR TITLE
corrections from previous PR cockup

### PR DESCRIPTION
**Pull request by astroyouth**
previous corrections to single line translate PR done here.
DONT MERGE YET FRED!!

### CODE REVIEW

File: my-video-room/library/class-shortcodedocuments.php [hunk 1]
```diff
@@ -51,16 +51,21 @@ public function render_general_shortcode_docs() {
 			);
 			?>
 			<br />
+
 			<?php
-				echo esc_html__(
-					'Room=(one of the following - meet-center, bookings-center, site-video-room) - selects the auto generated room type to query. This is required.<br>Type (title) - Room Name (with spaces) - Type (slug) - returns the post slug (eg- www.c.com/jones has slug of Jones) - Type (post_id) - returns the WordPress Post ID of a room Type (url) - returns URL of room. <br>Usage - ',
+			\sprintf(
```
Comment (alecsammon) [inline my-video-room/library/class-shortcodedocuments.php:56]: you need `\printf` here

File: my-video-room/library/class-shortcodedocuments.php [hunk 2]
```diff
@@ -51,16 +51,21 @@ public function render_general_shortcode_docs() {
...
+				/* translators: %s is the site URL */
+				esc_html__(
+					'Room=(one of the following - meet-center, bookings-center, site-video-room) - selects the auto generated room type to query. This is required.<br>Type (title) - Room Name (with spaces) - Type (slug) - returns the post slug (eg- %s has slug of Jones) - Type (post_id) - returns the WordPress Post ID of a room Type (url) - returns URL of room. <br>Usage - ',
```
Comment (alecsammon) [inline my-video-room/library/class-shortcodedocuments.php:59]: I think the this one should be split in half at the `<br>`

Also - the `Usage -` should be moved the the next block

Comment (astroyouth) [reply my-video-room/library/class-shortcodedocuments.php:59]: done

File: my-video-room/library/class-shortcodedocuments.php [hunk 3]
```diff
@@ -51,16 +51,28 @@ public function render_general_shortcode_docs() {
...
 			?>
 			<br />
+
 			<?php
-				echo esc_html__(
-					'Room=(one of the following - meet-center, bookings-center, site-video-room) - selects the auto generated room type to query. This is required.<br>Type (title) - Room Name (with spaces) - Type (slug) - returns the post slug (eg- www.c.com/jones has slug of Jones) - Type (post_id) - returns the WordPress Post ID of a room Type (url) - returns URL of room. <br>Usage - ',
+			\printf(
+				/* translators: %s is the site URL */
+				esc_html__(
+					'Room=(one of the following - meet-center, bookings-center, site-video-room) - selects the auto generated room type to query. This is required.<br>Type (title) - Room Name (with spaces) - Type (slug) - returns the post slug (eg- %s has slug of Jones) - Type (post_id) - returns the WordPress Post ID of a room Type (url) - returns URL of room.',
```
Comment (alecsammon) [inline my-video-room/library/class-shortcodedocuments.php:59]: can we split at `<br>Type....` also?

File: my-video-room/library/class-shortcodedocuments.php [hunk 4]
```diff
@@ -51,16 +51,28 @@ public function render_general_shortcode_docs() {
...
+			);
+			?>
+			<br />
+			<?php
+			echo esc_html__(
+				'Usage -',
+				'my-video-room'
+			);
 			?>
 			<br />
```
Comment (alecsammon) [inline my-video-room/library/class-shortcodedocuments.php:72]: I don't think this `<br>` should be here

Comment (alecsammon) [reply my-video-room/library/class-shortcodedocuments.php:72]: Ah actually

File: my-video-room/library/class-shortcodedocuments.php (hunk 3 above)
Comment (astroyouth) [reply my-video-room/library/class-shortcodedocuments.php:59]: Done

**Review by alecsammon — APPROVED** [review]

**Pull request merged**